front/app/containers/Admin/projects/project/participationContext/components/ParticipationMethodPicker.tsx

```diff
@@ -182,7 +198,43 @@ export const ParticipationMethodPicker = ({
               }
             />
           )}
-          <FeatureFlag name="volunteering">
+          {documentAnnotationAllowed && (
+            <Tippy
+              maxWidth="250px"
+              placement="right-end"
+              content={formatMessage(messages.contactGovSuccessToAccess)}
+              // Don't show Tippy tooltip if the feature is enabled
+              disabled={documentAnnotationEnabled}
+              hideOnClick={false}
+            >
+              <div>
+                <ParticipationMethodRadio
+                  disabled={!documentAnnotationEnabled}
+                  onChange={handleParticipationMethodOnChange}
+                  currentValue={participation_method}
+                  value="document_annotation"
+                  name="participationmethod"
```

Comment:
The naming seems inconsistent but we can leave as is for now but it would be great to use something consistent. I see many areas in this file like that

Reply:
Just copy pasted here... Not going to look into this now.